drivers/entropy/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ zephyr_library_sources_ifdef(CONFIG_ENTROPY_MCUX_RNGA          entropy_mcux_rnga
 zephyr_library_sources_ifdef(CONFIG_ENTROPY_MCUX_TRNG          entropy_mcux_trng.c)
 zephyr_library_sources_ifdef(CONFIG_ENTROPY_NRF5_RNG           entropy_nrf5.c)
 zephyr_library_sources_ifdef(CONFIG_ENTROPY_SAM_RNG            entropy_sam.c)
+zephyr_library_sources_ifdef(CONFIG_ENTROPY_SIMPLELINK_RNG     entropy_simplelink.c)
```

Comment:
Is the entropy driver needed for base functionality?  if not I'd ask to pull it into a separate PR after we get the base SoC/board ports merged.

Reply:
No it is not essential. I will split it into a seperate PR. I included it after reading [here](https://docs.zephyrproject.org/latest/reference/drivers/index.html#standard-drivers) that all supported boards have a random number generator driver, but I think I misunderstood.